lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -778,14 +803,38 @@ def accessible_objects(user_or_attribute, operation, options = {})
       end
     end
 
+    # Returns a map of operation names to the user_or_attribute's accessible objects via each operation
+    def accessible_objects_for_operations(user_or_attribute, operations, options = {})
+      unless options[:direct_only]
+        raise ArgumentError, 'Functionality for indirect objects is not yet implemented!'
```

Comment:
do you mean `ArgumentError` or `NoMethodError`? either way im fine as is w `ArgumentError`

Reply:
i mean in `lib/policy_machine.rb` i went with `NoMethodError` for when this function `accessible_objects_for_operations` is not implemented on an adapter just because it is used in the rest of that file, even though i think `NotImplementedError` would be more correct there.

`lib/policy_machine.rb` is basically a higher level wrapper than this active record adapter file we are in here.  it defines the functions of the `PolicyMachine` class and then tries to send them on whatever adapter is configured.  if the particular adapter does not have an implementation of the function, it raises `NoMethodError`.

Reply:
ah gotcha didn't realize you were referring to that other file, resolving